<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCommentDialog</class>
 <widget class="QDialog" name="addCommentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>235</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sewers Add Comment</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../sewersIcon.ico</normaloff>../sewersIcon.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="addCommentTitleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>195</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>195</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ADD COMMENT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="addCommentPlainTextEdit">
     <property name="placeholderText">
      <string>Comments are limited to 250 characters...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="addCommentButtonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addCommentButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>addCommentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addCommentButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>addCommentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
